### ISSUE TITLE
[Bug]: When the resolution is not 1024x1024, an error message is displayed.

### ISSUE THREAD

**Issue by zismylove** [huchenlei/ComfyUI-layerdiffuse#37]
### What happened?

When the input image resolution is not 1024x1024 at node  layerDiffusionDiffApply, an error message is displayed.
![image](https://github.com/huchenlei/ComfyUI-layerdiffuse/assets/11270289/c9f2ee31-4760-42d2-b539-1c938a727f48)
, use nodes to change the resolution by upscalelatent,the first error pass,but comming nex one,node layerDiffuseDecode report
![image](https://github.com/huchenlei/ComfyUI-layerdiffuse/assets/11270289/ec59f0e2-7c90-481b-8e3c-8c50ec8b4da2)
The resolution that reported an error is 1366x768.
I checked to make sure the resolutions match, but only 1024x1024 resolution works fine.

### Steps to reproduce the problem

/

### What should have happened?

/

### Commit where the problem happens

ComfyUI:
ComfyUI-layerdiffuse:

### Sysinfo

Error occurred when executing LayeredDiffusionDiffApply:

Sizes of tensors must match except in dimension 1. Expected size 170 but got size 171 for tensor number 1 in the list.

File "H:\diffusion\lcm\ComfyUI\ComfyUI\execution.py", line 152, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)

File "H:\diffusion\lcm\ComfyUI\ComfyUI\execution.py", line 82, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "H:\diffusion\lcm\ComfyUI\ComfyUI\execution.py", line 75, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
File "H:\diffusion\lcm\ComfyUI\ComfyUI\custom_nodes\ComfyUI-layerdiffusion\layered_diffusion.py", line 392, in apply_layered_diffusion
torch.cat([latent["samples"], blended_latent["samples"]], dim=1)
![image](https://github.com/huchenlei/ComfyUI-layerdiffuse/assets/11270289/3fe01c6a-588f-4fee-871e-421cc52b543b)

### Console logs

```Shell
/
```

### Workflow json file

/

**Comment by huchenlei** [comment]
Stable diffusion makes 8x compression from pixel space to latent space. So the input image's dimensions need to be multiple of 8. 8 x 170 = 1360.

**Comment by huchenlei** [comment]
I will add an explicit error message to make this clear.

**Comment by huchenlei** [comment]
Explicit error message added in https://github.com/huchenlei/ComfyUI-layerdiffuse/commit/74aee185528d76e41486400145596314aac1b957
